data/datacatalogue/CatalogueOntologies/Organisation roles.csv

```diff
@@ -1,7 +1,6 @@
-name
-Data originator
```

Comment:
Data originator is removed

Reply:
Hmm, that we should double check with Rosa because she uses that word a lot